internal/cephfs/fsjournal.go

```diff
@@ -232,7 +232,7 @@ func reserveVol(ctx context.Context, volOptions *volumeOptions, secret map[strin
 	imageUUID, vid.FsSubvolName, err = j.ReserveName(
 		ctx, volOptions.MetadataPool, util.InvalidPoolID,
 		volOptions.MetadataPool, util.InvalidPoolID, volOptions.RequestName,
-		volOptions.NamePrefix, "", "", volOptions.ReservedID)
+		volOptions.NamePrefix, "", "", volOptions.ReservedID, "")
```

Comment:
the flag `extra-create-metadata` is set for both cephfs and rbd deployment above but we are not setting the owner for cephfs is there any reason for that one?

Reply:
It has been added to have the options in sync as much as possible. CephFS does not set the KMS-ID either yet, at that point the Owner can be added too.